sql/telemetry_derived/fxa_users_services_first_seen_v1/init.sql

```diff
@@ -44,12 +45,23 @@ WITH
     AND service IS NOT NULL
     AND user_id IS NOT NULL
   WINDOW
-    w1 AS (
+    -- We must provide a window with `ORDER BY timestamp DESC` so that udf_mode_last actually aggregates mode first.
+    w1_reversed AS (
       PARTITION BY
         user_id,
         service
       ORDER BY
-        `timestamp` ) ),
+        `timestamp` DESC
+      ROWS BETWEEN
+        UNBOUNDED PRECEDING
+        AND UNBOUNDED FOLLOWING),
```

Comment:
this was missing a frame, preventing the window from aggregating more than one row (because the default frame is `ROWS UNBOUNDED PRECEDING`)